src/test/goldens/paper-button/paper-button.d.ts

```diff
@@ -9,9 +9,7 @@
  */
 
 /// <reference path="../polymer/types/polymer.d.ts" />
-/// <reference path="../iron-flex-layout/iron-flex-layout.d.ts" />
 /// <reference path="../paper-behaviors/paper-button-behavior.d.ts" />
-/// <reference path="../paper-styles/element-styles/paper-material-styles.d.ts" />
```

Comment:
Is this necessary, or just a style/legibility improvement? 

Or, put another way, do we generate empty `.d.ts` files, or do we leave them out if they're empty?

Reply:
A missing reference will cause tsc to error. We generally do generate empty `d.ts` files, but in these cases Analyzer doesn't even return a document for the HTML file, because they only contain CSS. It's also just kind of odd IMO to emit declarations for files that have no JS footprint, so my plan is to just not run the generator on these style-only repos.

Reply:
Discussed offline, we agreed to emit `d.ts` files even for HTML files with no script. Removed the commit that added the default `removeReferences`, and will add this new behavior in a new PR.